IBPSA/Fluid/HeatExchangers/BaseClasses/PartialEffectiveness.mo

```diff
@@ -83,7 +83,7 @@ equation
            fra_a2 * Medium2.specificHeatCapacityCp(state_a2_inflow) +
            fra_b2 * Medium2.specificHeatCapacityCp(state_b2_inflow) else
         Medium2.specificHeatCapacityCp(state_a2_inflow));
-  CMin_flow = min(C1_flow, C2_flow);
+  CMin_flow = IBPSA.Utilities.Math.Functions.smoothMin(x1=C1_flow, x2=C2_flow, deltaX=CMin_flow_small);
```

Comment:
I am not convinced that this is not giving large spikes if C1_flow = 0 and C2_flow transitions through 0. Then, you can get an undershoot of min(C1_flow, C2_flow), leading to a large ratio of CMin_flow/min(C1_flow, C2_flow), and therefore large QMax_flow/min(C1_flow, C2_flow), which in turn leads to large outlet temperatures near zero flow. This is why I refrained from using `smoothMin` but rather used `min`.